index.html

```diff
@@ -75,79 +81,88 @@ <h2>Starting with a staircase</h2>
                 </div>
-                <div>
-                    <span>"</span>
+                <div class="quotes-rectangle">
+                    <span class="quotes">"</span>
+                    <div class="rectangle"></div>
                     <blockquote>
-                        Some children could creatively enjoy inventing functions
+                        Some children could creatively<br>enjoy inventing functions
                     </blockquote>
                 </div>
-                <div>
-                    <div>
+                <div  class="paragraphs">
+                    <div class="block-one">
                         <h2>Programming</h2>
                         <p>Very soon the children were able to construct their own rod calculator in Haskell, by first
-                            constructing the engine using the <span>Colour</span> data type. Then the children asked the
+                            constructing the engine using the <span class="highlighted-words">Colour</span> data type. Then the children asked the
                             same role-play questions, this time by typing the language into the interpreter window. It
                             soon
                             became apparent that errors in the engine meant that the interpreter would not</p>
                     </div>
-                    <p>number names for the rods, measured with White rather than the colour names. He becomes aware as
-                        he is speaking that the version of the <span>succ</span> function that works for a staircase
-                        with a step size White is distinct from the version of <span>succ</span> in an even staircase,
+                    <p class="block-two">number names for the rods, measured with White rather than the colour names. He becomes aware as
+                        he is speaking that the version of the <span class="highlighted-words">succ</span> function that works for a staircase
+                        with a step size White is distinct from the version of <span class="highlighted-words">succ</span> in an even staircase,
                         and he corrects himself. It is clear to see that he has constructed an image of the staircase
                         and is virtually traversing it.</p>
                 </div>
                 <figure>
-                    <figcaption>Transcript of a conversation about program behaviour.</figcaption>
-                    <img src="img/2.jpg" alt="math">
+                    <figcaption>
+                        <ul>
+                            <li><span class="figcaption">Transcript of a conversation about program behaviour.</span></li>
+                        </ul>
+                    </figcaption>
+                    <img src="img/2.jpg" width="630" height="200" alt="math">
                 </figure>
             </div>
-            <div>
+            <div class="block-aside">
                 <aside>

```

Comment:
First of all - well done! Now it really looks alike the mockup, happy to see it! :)

From visual point of view I'll ask you to fix aside block:
1) please don't set height + width to blocks with text - usually we control it's size by selecting specific font size, line-height and paddings/margins. In your case you see that some of text doesn't fit the size of block - it's not what we want, don't we ? :)


2) Please add paddings to this block - you don't have any at the moment
![image](https://user-images.githubusercontent.com/104353194/169474861-13ef485d-5302-4829-8d7e-d2b8e8903a4c.png)